<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KarteAdresseAnlegen</class>
 <widget class="QDialog" name="KarteAdresseAnlegen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="Laengengrad">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Laengengrad</string>
   </property>
  </widget>
  <widget class="QLabel" name="Breitengrad">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Breitengrad</string>
   </property>
  </widget>
  <widget class="QLabel" name="Strasse">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>57</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Strasse</string>
   </property>
  </widget>
  <widget class="QLabel" name="Hausnummer">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Hausnummer</string>
   </property>
  </widget>
  <widget class="QLabel" name="Postleitzahl">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Postleitzahl</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Bestaetigen">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>250</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Bestaetigen</string>
   </property>
  </widget>
  <widget class="QLabel" name="Stadt">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Stadt</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LaengengradLine">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="BreitengradLine">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="StrasseLine">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="HausnummerLine">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>140</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="PostleitzahlLine">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>170</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="StadtLine">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>200</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
